### DISCUSSION TITLE
Raspistill vs MotionPlus performance

### DISCUSSION THREAD

**Discussion by saharco540** [Motion-Project/motionplus#98] [Q&A]
Hey,
Place my RPI 2 A+ with RPi Camera outside and noticed over-exposed images in the motion triggered and snapshots.
Using raspistill with no special configuration gets great, not over-exposed images.
How can I address this?
Thanks

**Comment by Mr-Dave** [comment]
I assume you are using the libcamera feature of Motionplus.  I'd say first just run as normal motion detection in Motion rather than just triggered snapshots to see if it is something related to only doing a single snapshot.  Next, libcamera has many [configuration options](https://motion-project.github.io/motionplus_config.html#libcam_params) to try.  Motionplus doesn't change any of these by default.
